migrations/migrations2.sql

```diff
@@ -0,0 +1,3 @@
+ALTER TABLE IF EXISTS courses (
+    ADD CONSTRAINT fk_courseId FOREIGN KEY (courseId) REFERENCES reviews(courseId)
+)
```

Comment:
I believe this relationship is backwards: each review should have a foreign key courseId that references the id field of a given course.